src/components/views/Sidebar.js

```diff
@@ -218,14 +219,14 @@ export default function Sidebar({view, children}) {
                     <SidebarItem
                         open={open}
                         text="Login"
-                        onClick={() => {history.push('/login')}}
-                        icon={<LoginIcon />}   
+                        onClick={() => {history.push('/login'); localStorage.removeItem('errorMessage');}}
```

Comment:
Should this be a fix to remove the error message from when the username or password is incorrect? That is just a quick hack to fix the underlying problem: that you haven't used hooks for the error message. I will make a PR into your branch that fixes that, you can review it and if you're happy with it, pull it into your branch. (then you won't need what you added here).

Reply:
Merged your PR thanks :)